niworkflows/viz/plots.py

```diff
@@ -770,3 +911,28 @@ def confounds_correlation_plot(confounds_file, output_file=None, figure=None,
         figure = None
         return output_file
     return [ax0, ax1], gs
+
+
+def _get_tr(img, is_cifti=False):
```

Comment:
Let's go more pythonic EAFP:
```suggestion
def _get_tr(data):
```